### PR TITLE
dbt-dremio 1.3.0 Release Readiness

### PR DESCRIPTION
**Pull request by jlarue26**
### Summary

dbt-dremio 1.3.0 Release Readiness

### Description

Update:
- README
- versioning
- dev_requirements
- THIRD_PARTY_LICENCES

### Related Issue

None

### Additional Reviewers

@ArgusLi 
@ravjotbrar

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,18 +15,22 @@ Installing the dbt-dremio package will install or update dbt-core to version 1.3
...
 - Read more on the [dbt Community Discourse](https://discourse.getdbt.com)
 
 ## Reporting bugs and contributing code
-
-- Want to report a bug or request a feature? Let us know by opening [an issue](https://github.com/dremio/dbt-dremio/issues/new)
-
-
+---
+- Open bugs and feature requests can be found at [dbt-dremio's GitHub issues](https://github.com/dremio/dbt-dremio/issues).
+- Want to report a bug or request a feature? Let us know by on [Slack]([Slack](http://community.getdbt.com/)), or opening [an issue](https://github.com/dremio/dbt-dremio/issues/new)
```
Comment (ArgusLi) [inline README.md:31]: We do have a link to the slack channel now https://getdbt.slack.com/archives/C049G61TKBK

File: dev_requirements.txt [hunk 2]
```diff
@@ -17,4 +17,8 @@ pytest-xdist
 pytz
 tox>=3.13
 twine
-wheel
\ No newline at end of file
+wheel
+agate==1.6.3
+requests==2.28.1
+setuptools==63.2.0
+third-party-license-file-generator
```
Comment (ArgusLi) [inline dev_requirements.txt:24]: why do we need this? I don't think we used this at all.

File: README.md (hunk 1 above)
Comment (jlarue26) [reply README.md:31]: I followed the example from Snowflake where it just went to the general dbt community Slack, but can add the link to ours.

File: dev_requirements.txt (hunk 2 above)
Comment (jlarue26) [reply dev_requirements.txt:24]: Yeah, accidentally got pulled in with one of the license generator tools I was using. I will remove it.

**Pull request merged**